### DISCUSSION TITLE
Autotile to Nearest [feature request]

### DISCUSSION THREAD

**Discussion by tsbertalan** [gTile/gTile#392] [Ideas]
A nice feature would be if I could approximately lay out my windows as I want (with Ubuntu's alt+ right click, most likely), and then use some shortcut to snap them all to the nearest grid lines for the currently set grid resolution. (And maybe don't be so literal about "nearest", if it would leave an extra row or column of space between two windows.)

**Comment by scherepanov** [comment]
Cool idea!
Submit PR!
